src/page-modules/assistant/server/journey-planner/index.ts

```diff
@@ -349,10 +480,22 @@ function inputToLocation(
     name: input[direction].name,
   };
 }
+function inputToViaLocation(input: TripViaInput, direction: 'via') {
+  return {
+    place: input[direction].id,
+    coordinates: {
+      latitude: input[direction].geometry.coordinates[1],
+      longitude: input[direction].geometry.coordinates[0],
+    },
+    name: input[direction].name,
+    minSlack: 'PT120S',
```

Comment:
Should maybe be a params instead of a static values?

Reply:
If this is default and never changing I would move it to the graphql query document instead.

Reply:
I agree with Morten.